### ISSUE TITLE
I need to extend the code that grabs streaming replies to higher order decedents SO THAT a full tree can be built dynamically

### ISSUE THREAD

**Issue by scramblingbalam** [scramblingbalam/Alta_Real#41]

**Comment by scramblingbalam** [comment]
This is dependent on #61 and I will use the code from that issue to complete this task

**Comment by scramblingbalam** [comment]
It should be noted that I'm not sure that collecting order>1 replies is possible vie streaming as it would mean altering the stream in progress.
If it proves impossible than a browser automation method will be tried using Selenium's python API

**Comment by scramblingbalam** [comment]
From my research it seems impossible to gather higher order replies that where `@realDonaldTrump` has been removed from the conversation via the streaming api. As seen h ere: [http://stackoverflow.com/questions/13215054/tweepy-public-stream-filter-by-a-changing-variable](url)
because in order to capture replies the stream would have to start listening immediately disconnection isn't possible.
Instead the best option is to use the search api to find them  latter.
This however is difficult and so I've decided to try and just use the deep tweets that mention `realDonaldTrump` this is consistent with the way that tweets where collected by Zeb since he only captured tweets that used as specific hashtag
